tests/profiling/collector/test_stack.py

```diff
@@ -745,7 +745,7 @@ def _nothing():
         if event.task_name == "MainThread":
             main_thread_found = True
         elif event.task_id in {t.ident for t in threads}:
-            for filename, lineno, funcname, classname in event.frames:
+            for _filename, _lineno, funcname, _classname in event.frames:
```

Comment:
What is the problem here?

Reply:
Another instance of `B007: Loop control variable not used within the loop body. If this is intended, start the name with an underscore.`